### ISSUE TITLE
Word 2013: Ribbons not announced when accessed via shortcut keys

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4683]
**Reported by netblue on 2014-12-11 14:40**
High all, when moving to the file menus or ribbons such as insert, design, start  and others in word 2013 with the short cut keys NVDA does not announce the name of the menu, and it only says the elements when the arrow keys are pressed. I made a fresh install of office 2013 to make shore that it was not my configurations of word are the culprits. Also note that in PowerPoint an excel Nvda announces the ribbon names when the short cut keys are used.

**Comment by bhavyashah** [comment]
Using Word 2010 and NVDA 2017.3 on a Windows 8.1 system, I was unable to reproduce the reported issue via the following STR:
* Open a blank Word document.
* Press Alt+N to navigate to the Insert Tab.
Expected and Actual Result: NVDA reports the Insert Tab as part of the Word ribbon along with its keyboard shortcut, i.e. "Ribbon Tabs  tab control
Insert  tab  selected  Alt, N"

Since the ticket body explicitly mentions Insert (I assume Tab) as a test case, something that is unable to yield any odd results at my end, I can safely say that if my inference of the reporter's description of the issue is correct, then this bug probably no longer exists. However, there are two things that need to be noted before we consider closing this ticket:
* This ticket was originally reported with Word 2013, while my tests have been conducted with Word 2010. It is important that we double-check the reported issue with the version of the software this ticket was actually filed with regards to.
* I am simply assuming, out of generality, that Alt+F takes a user to the File menu in Word just like it does in a standard application. However, I do recognize that Word has ribbons and lacks the standard menu bar. Regardless, if Alt+F is indeed taking me to a File menu in Word, then I must report that upon pressing Alt+F in a blank Word document in case of Word 2010, NVDA says speaks "Save  Save  Alt, F, S" without acknowledgeing any File menu or menu/ribbon at all. Is this a valid issue and in actuality what netblue  was trying to convey? CC @Qchristensen

**Comment by Qchristensen** [comment]
Ok, I can confirm in Office 2007, 2013 and 2016 that the Insert ribbon is identified when you press alt+n, as are other ribbons.

The "File menu" has been an odd beast in Office since they brought ribbons in.  In Office 2007, there is no "file" label - everyone just called it that.  Visually it is a circle with the "office logo" and navigating to it (eg press alt, then left arrow) reports it as "office button".  In later versions of Word, it is back to a text label reading "file", although this isn't announced when you press alt+f, and in Office 2016, when you get into the file menu, it becomes full screen and the original "file" ribbon label isn't visible at all.

My inclination is to close this as the original issue of the insert etc ribbons not reading is now working for everyone.  The file menu is a bit different, but then it is for everyone.
